spec/support/pages/authentication.rb

```diff
@@ -6,6 +6,11 @@ def login(user, password="password")
       login_page.visit_page.login(user, password)
     end
 
+    def login_with_google(user, password="password")
+      login_page = Pages::LoginPage.new
+      login_page.visit_google_login
+    end
```

Comment:
matching the patterns, it feels like this line should look something like `login_page.visit_page.login_with_google`